client/components/plans/plan-price/index.jsx

```diff
@@ -24,10 +23,8 @@ const PlanPrice = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
-			if ( abtest( 'planPricing' ) === 'monthly' ) {
-				const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
-				formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
-			}
+			const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
```

Comment:
I see that the `+` was already here, but since we're calling `.toFixed()`, what purpose is it serving?